engines/titanic/star_control/star_closeup.cpp

```diff
@@ -174,9 +175,8 @@ bool CStarCloseup::setup2(int val1, int val2) {
 				e->_pixel2 = (val >> 8) & 0xff;
 				e->_pixel3 = (val >> 16) & 0xff;
 				e->_field8 = g_vm->getRandomNumber(3) + 3;
-
 				e->_fieldC = g_vm->getRandomNumber(255);
-				e->_field10 = FACTOR * (float)g_vm->getRandomNumber(15);
+				e->_field10 = Common::deg2rad<double>((double)g_vm->getRandomNumber(15));
```

Comment:
I see this one was changed from a single precision floating point arithmetic to double precision, but I can see why you did it. Also since this is on a random number, it is unlikely to break anything.

The whole file is a bit strange. The `_field10` is a double as far as I can see, it was using a float `FACTOR`, but all the floating point literal values were double. So it ended up doing double precision arithmetic, but with limited precision on the factor. This means your changes might result in a slightly different value for those as well, but it looks like that is what was intended. @dreammaster might want to check this though.